praktikumsplaner-frontend/frontend/src/api/PraktikumsstellenService.ts

```diff
@@ -121,4 +121,39 @@ export default {
             }
         });
     },
+    deletePraktikumsstelle(stelle: Praktikumsstelle): Promise<void> {
+        if (isAusbildunsPraktikumsstelle(stelle)) {
+            return fetch(
+                `${API_BASE}${PRAKTIKUMSSTELLE_BASE}/ausbildung/${stelle.id}`,
+                FetchUtils.getDELETEConfig({})
+            ).then((response) => {
+                FetchUtils.defaultResponseHandler(response);
+                useSnackbarStore().showMessage({
+                    message: "☑ Praktikumsstelle erfolgreich gelöscht",
+                    level: Levels.SUCCESS,
+                });
+            });
+        } else if (isStudiumsPraktikumsstelle(stelle)) {
+            return fetch(
+                `${API_BASE}${PRAKTIKUMSSTELLE_BASE}/studium/${stelle.id}`,
+                FetchUtils.getDELETEConfig({})
+            ).then((response) => {
+                FetchUtils.defaultResponseHandler(response);
+                useSnackbarStore().showMessage({
+                    message: "☑ Praktikumsstelle erfolgreich gelöscht",
+                    level: Levels.SUCCESS,
+                });
+            });
+        } else {
+            throw new Error("Praktikumsstelle nicht gefunden.");
```

Comment:
that message doesnt fits its trigger. It is not know if the Stelle exits. In that cause, when the error is thrown, it is unknown which kind of stelle it is